include/crow/http_server.h

```diff
@@ -213,6 +221,7 @@ namespace crow
         std::vector<std::unique_ptr<asio::io_service>> io_service_pool_;
         std::vector<detail::task_timer*> task_timer_pool_;
         std::vector<std::function<std::string()>> get_cached_date_str_pool_;
+        std::vector<int> task_queue_length_pool_;
```

Comment:
might be best to use `unsigned int` instead of `int` since the length cannot go below 0.

Reply:
Done :)